src/main/java/de/tum/in/www1/artemis/web/rest/ModelingSubmissionResource.java

```diff
@@ -185,7 +185,7 @@ public ModelingSubmissionResource(SubmissionRepository submissionRepository, Res
      *         found
      */
     @GetMapping("/modeling-submissions/{submissionId}")
```

Comment:
This REST Controller validates multiple rules we have in our documentation: https://docs.artemis.ase.in.tum.de/dev/guidelines/server/#keep-your-restcontrollers-clean-and-focused
E.g. the naming of the mappings: here it should be "modeling-submissions/{submissionId}" without the "/" at the beginning, etc.

I know that this is not really your tasks but I think that this can be rather quickly adjusted. (It is very important for Stephan that the REST Controllers follow our documentation very closely)
Mabye this does not affect your PR but definetly make sure if you add any new method that it follows the documentation to a T